package.json

```diff
@@ -60,8 +60,9 @@
     "eslint-plugin-react-hooks": "4.6.0",
     "husky": "^7.0.1",
     "jest-styled-components": "^7.0.5",
+    "lint-staged": ">=10",
```

Comment:
Why not use the rule same as other dependencies, e.g. `"lint-staged": "^13.2.3",`

Reply:
It's automatically added by `npx mrm@2 lint-staged`, but seems not the best version. I've updated it.